eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/LightningMessageCodecs.scala

```diff
@@ -476,6 +480,7 @@ object LightningMessageCodecs {
     .typecase(37000, spliceInitCodec)
     .typecase(37002, spliceAckCodec)
     .typecase(37004, spliceLockedCodec)
+    .typecase(37006, stfuCodec)
```

Comment:
Same here, no need to use a custom message type, we should directly use the one from the spec PR.

Reply:
Rusty chose `2`? I so we're now back filling the early bits?

Fixed in [9fa905a](https://github.com/ACINQ/eclair/pull/2680/commits/9fa905a3e557420b011b8c5079cb48ba74b705a2)